Matching/cps_rets.py

```diff
@@ -134,15 +144,588 @@ def create(self, record):
         """
         unit = OrderedDict()
         self.nunits += 1
+        #flag this person
+        record['flag'] = True
+        #income items
+        was = record['wsal_val']
+        wasp = was
+        intst = record['int_val']
+        dbe = record['div_val']
+
+        alimony = record['alm_val']
+        bil = record['semp_val']
+        pensions = record['rtm_val']
+        rents = record['rnt_val']
+        fil = record['frse_val']
+        ucomp = record['uc_val']
+        socsec = record['ss_val']
+        #weight & flags
+        wt = record['fsup_wgt']
+        ifdept = record['d_flag']  # Tax unit dependent flag
+        record['h_flag'] = True   # Tax Unit Head Flag
+        # CPS identifiers
+        xhid = record['h_seq']
+        xfid = record['ffpos']
+        xpid = record['ph_seq']
+        xstate = record['gestfips']
+        xregion = record['gereg']
+        # CPS evaluation criteria (head)
+        zifdep = record['d_flag']   # Tax unit dependent flag
+        zntdep = 0
+        zhhinc = record['hhinc']
+        zagept = record['a_age']
+        zagesp = 0
+        zoldes = 0
+        zyoung = 0
+        zworkc = record['wc_val']
+        zsocse = record['ss_val']
+        zssinc = record['ssi_val']
+        zpubas = record['paw_val']
+        zvetbe = record['vet_val']
+        zchsup = 0
+        zfinas = 0
+        zdepin = 0
+        zowner = 0
+        zwaspt = record['wsal_val']
+        zwassp = 0
+
+        #home Ownership Flag
+        if (self.nunits == 1) and (record['h_tenure'] == 1):
+            zowner = 1
+
+        # store dependents info
+        for i in range(1,17):
+            record['dep' + str(i)] = np.nan
+        for i in range(1,17):
+            record['depage' + str(i)] = np.nan
+
+        # marital status
+        ms = record['a_maritl']
+        if ms == 1 or ms == 2 or ms == 3:
+            type = 2
```

Comment:
@XueliangWang, you need to change `type` to something else. `type` is a built-in function in python so it shouldn't be used as a variable name.